ci-operator/jobs/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main-presubmits.yaml

```diff
@@ -64,7 +64,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-stolostron-multicluster-global-hub-main-sonarcloud
     rerun_command: /test sonarcloud
-    skip_if_only_changed: ^doc/|^samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
+    skip_if_only_changed: ^doc/|^samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.py$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
     spec:
```

Comment:
we don't need `.md` any more?

Reply:
The `.md` is still here. We wrote it repeatedly.